composer.json

```
@@ -5,16 +5,16 @@
         "psr-4": { "M6Web\\Bundle\\": "src/" }
     },
     "require" : {
-        "php": "~7",
+        "php": ">=7.2",
         "ext-amqp": "*",
         "symfony/dependency-injection": "^3.4 || ^4.3 || ^5.0",
-        "symfony/framework-bundle": "^3.4 || ^4.3",
-        "symfony/http-kernel": "^3.4 || ^4.3",
+        "symfony/framework-bundle": "^3.4 || ^4.3 || ^5.0",
+        "symfony/http-kernel": "^3.4 || ^4.3 || ^5.0",
         "symfony/yaml": "^3.4 || ^4.3 || ^5.0",
         "twig/twig": "^1.31 || ^2.0 || ^3.0"
     },
     "require-dev" : {
-        "atoum/atoum": "^3.4.1",
+        "atoum/atoum": "~3.4||~4.0",
```

```suggestion
        "atoum/atoum": "~4.0",
```

if we drop `< php7.4`